modules/features/player/src/main/res/layout/fragment_player_container.xml

```diff
@@ -9,74 +9,87 @@
+                <com.google.android.material.tabs.TabLayout
+                    android:id="@+id/tabLayout"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:background="@color/transparent"
+                    style="@style/Widget.MaterialComponents.TabLayout"
+                    app:tabIndicatorFullWidth="false"
+                    app:tabTextAppearance="@style/PlayerTabTextAppearance"
+                    app:tabGravity="fill"
+                    app:tabMode="scrollable"
+                    app:tabIndicatorColor="?attr/player_contrast_01"
+                    app:tabTextColor="?attr/player_contrast_02"
+                    app:tabSelectedTextColor="?attr/player_contrast_01"
+                    app:tabBackground="@color/transparent"
+                    app:tabPaddingStart="@dimen/player_tab_padding"
+                    app:tabPaddingEnd="@dimen/player_tab_padding"
+                    app:tabMinWidth="0dp" />
+                <FrameLayout
+                    android:layout_width="68dp"
+                    android:layout_height="match_parent">
+                    <ImageView
+                        android:id="@+id/upNextButton"
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:background="?attr/selectableItemBackgroundBorderless"
+                        android:layout_gravity="center"
+                        app:srcCompat="@drawable/mini_player_upnext_badge"
+                        app:tint="?attr/player_contrast_01" />
+                    <TextView
+                        android:id="@+id/countText"
+                        android:layout_width="19dp"
+                        android:layout_height="19dp"
+                        android:textStyle="bold"
+                        android:layout_marginEnd="30.5dp"
+                        app:autoSizeTextType="uniform"
+                        app:autoSizeMinTextSize="8sp"
+                        app:autoSizeMaxTextSize="16sp"
+                        app:autoSizeStepGranularity="1sp"
+                        android:letterSpacing="-0.05"
+                        android:fontFamily="sans-serif-medium"
+                        android:gravity="center"
+                        android:layout_gravity="end|center_vertical"
+                        android:textColor="#000" />
+                </FrameLayout>
+
+            </LinearLayout>
+
+            <au.com.shiftyjelly.pocketcasts.views.multiselect.MultiSelectToolbar
+                android:id="@+id/multiSelectToolbar"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:minHeight="?android:attr/actionBarSize"
+                android:theme="@style/ThemeDark"
+                android:visibility="gone"/>
```

Comment:
Reuses existing `MultiSelectToolbar` `Android` view. Thankfully, it interoperates well with the bookmarks `Compose` view. 😅